variables.tf

```diff
@@ -411,3 +411,11 @@ variable "sms_sender" {
 variable "sms_region" {
   default = ""
 }
+variable "agency_logo_path" {
+  description = "Path for the agency logo file - will be uploaded to s3"
+  default     = ""
+}
+variable "agency_logo_s3_key" {
```

Comment:
Any reason we need this - why would we need to vary ?

Reply:
not at all.